### PR TITLE
match angular CLI

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -106,8 +106,9 @@ nodejs_image_repos()
 
 git_repository(
     name = "io_bazel_rules_k8s",
+    commit = "d72491924369868a13532af9f15cb766e5943855",
     remote = "https://github.com/bazelbuild/rules_k8s.git",
-    tag = "v0.1",
+    shallow_since = "1545317854 -0500",
```
Comment (gregmagolan) [inline WORKSPACE]: neat

File: src/lib/shorten/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,12 @@
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+
+package(default_visibility = ["//:__subpackages__"])
+
+ts_library(
+    name = "shorten",
+    srcs = ["shorten.ts"],
+    module_name = "@bazel/shorten",
```
Comment (gregmagolan) [inline src/lib/shorten/BUILD.bazel]: A bit confusing using `@bazel/shorten` at the import site. Better name?

Comment (gregmagolan) [reply src/lib/shorten/BUILD.bazel]: Oh. Was the intention to also publish to @bazel/shorten?

Comment (alexeagle) [reply src/lib/shorten/BUILD.bazel]: yes, I think we should publish some dummy example library so we demonstrate how you'd share some code with a non-bazel part of your org.

**Pull request merged**